Add debian and ubuntu supported versions (#7331)
wazuh/wazuh-documentation: source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst

Installation on Debian/Ubuntu
-----------------------------

> +----------------------------------------------------------------------------+--------------------------+
> | The releases supported by the manifest to install Wazuh are as follows:                               |
> +------------+---------+--------+---------+--------+----------+----------+--------+-------------+-------+
> | **Ubuntu** | precise | trusty | vivid   | wily   | xenial   | yakketi  | bionic | focal fossa | jammy |
> +------------+---------+--------+---------+--------+----------+----------+--------+-------------+-------+
> | **Debian** | jessie  | wheezy | stretch | buster | bullseye | bookworm |                              |
> +------------+---------+--------+---------+--------+----------+----------+------------------------------+

Reviewer: ```suggestion
The manifest supports the following releases to install wazuh.

**Ubuntu**

-  Jammy Jellyfish
-  Focal Fossa
-  Bionic Beaver
-  Yakkety Yak
-  Xenial Xerus
-  Wily Werewolf
-  Vivid Vervet
-  Trusty Tahr
-  Precise Pangolin

**Debian**

-  bookworm
-  bullseye
-  buster
-  stretch
-  jessie
-  wheezy
```

Author: done
![image](https://github.com/wazuh/wazuh-documentation/assets/56500015/21fae729-6b32-42ad-8672-c02c8f42896e)

> .. note::

Reviewer: ```suggestion
```

Author: Done
![image](https://github.com/wazuh/wazuh-documentation/assets/56500015/3b13613e-a80e-40be-97f4-71ff5226bba9)

>   ``Ubuntu focal fossa`` is supported since version 4.0.4
> 
> 

Reviewer: ```suggestion
```
